Public/Src/Tools/SandboxExec/Program.cs

```diff
@@ -374,7 +374,8 @@ public List<string> DedupeAccessReports(params ISet<ReportedFileAccess>[] access
         /// <returns>SandboxedProcessInfo object that is configured to explicitly report all observed file accesses</returns>
         public static SandboxedProcessInfo CreateSandboxedProcessInfo(string processFileName, SandboxExecRunner instance)
         {
-            var sandboxProcessInfo = new SandboxedProcessInfo(fileStorage: instance, fileName: processFileName, disableConHostSharing: true, sandboxConnection: instance.m_sandboxConnection);
+            var sandboxProcessInfo = new SandboxedProcessInfo(
+                new PathTable(), fileStorage: instance, fileName: processFileName, disableConHostSharing: true, sandboxConnection: instance.m_sandboxConnection);

```

Comment:
>new PathTable() [](start = 16, length = 15)

not getting the whole context yet, but the fact that you are creating a new path table sounds fishy. Can't you use the single path table that always floats around?

Reply:
Ok, ignore my comment above, just saw that you just moved existing code around

---
In reply to: [334109096](https://github.com/microsoft/BuildXL/pull/1029#discussion_r334109096) [](ancestors = 334109096)

Reply:
exactly